CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol

```diff
@@ -0,0 +1,30 @@
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract VulnerableBankOther {
+  
+    using SafeMath for uint256;
+    mapping(address => uint) public balances;
```

Comment:
uint should be replaced with uint256 (also in the other contracts).

Reply:
as I said, this is their original code. Should I fix that?